src/logic/operate.test.js

```diff
@@ -0,0 +1,7 @@
+import operate from './operate';
+// import { test, expect } from '@testing-library/jest-dom'
+
+it('should be one of the operators and should not be string', () => {
+  expect(operate('1', '4', '-')).toMatch(/.+/i);
+  expect(operate('8', '4', '-')).not.toMatch(/a-zA-Z/i);
+});
```

Comment:
- [ ] Kindly add to your `operate.test.js` some tests for addition, multiple, modules, division, and subtraction. 

For example:
`expect(operate(10, 2, '÷')).toBe('5');`